src/Widgets/Others/Help.js

```diff
@@ -3,6 +3,8 @@
 //Components
 import { ModuleView } from '../Components/ModuleView/ModuleView';
 
+import jQuery from 'jquery';
```

Comment:
I was told that jQuery was only required for applications and not for packages.
When is it a good practice to use JQuery in a package ?

Reply:
Lesson learned: JQuery usage indicates the dependency towards some server. Thus code making such usage could be placed in the Extensions sub-directory in order to indicate this dependency.